openage/opus/opusenc.pyx

```diff
@@ -0,0 +1,415 @@
+# Copyright 2016-2017 the openage authors. See copying.md for legal info.
+
+#!python
+#cython: language_level=3
+#^- asdf: remove if unnecessary?
+
+import time
+from libc.string cimport memcpy, memset
+from cpython.mem cimport PyMem_Malloc, PyMem_Free
+from cpython.bytes cimport PyBytes_FromStringAndSize
+
+from ogg cimport *
+from opus cimport *
```

Comment:
we should avoid `*`-imports as they are very indirect and confusing where which functions comes from. I'd suggest to import them as `ogg` and `opus`, the 4 or 5 letters more won't hurt.